model/postgres/postgres_server.rb

```diff
@@ -246,6 +247,17 @@ def self.run_query(vm, query)
   def run_query(query)
     self.class.run_query(vm, query)
   end
+
+  def metrics_config
+    {
+      endpoints: [
+        "https://localhost:9090/federate?match[]=%7B__name__%3D~%22.%2B%22%7D"
```

Comment:
Just out of curiosity, what is `%7B__name__%3D~%22.%2B%22%7D`?

Reply:
This is urlencoded from `match[]={__name__=~".+"}`, which means to match all metrics to federate from the target prometheus